24life/scripts/delayed.js

```diff
@@ -1,7 +1,20 @@
-// eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { loadScript, sampleRUM } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/**
+ * Load the launch library applicable to the domain
+ */
+async function loadAdobeLaunch() {
+  const isProd = window.location.hostname === 'https://www.24hourfitness.com/';
+  const launchProd = 'https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN899f16b777754991924d45661d0c60bb.min.js';
+  const launchStaging = 'https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN1720538a14c5448383dbc3a400a4f55a-staging.min.js';
+  await loadScript(isProd ? launchProd : launchStaging, {
+    type: 'text/javascript',
+    charset: 'UTF-8',
+    async: true,
+  });
+}
+
```

Comment:
Dev: `"https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN95eda61e2ae9436886ce63d7d3dcb671-development.min.js"`

Staging:` "https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN1720538a14c5448383dbc3a400a4f55a-staging.min.js"`


Production: `"https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN899f16b777754991924d45661d0c60bb.min.js"`

Reply:
@synox  

- Could you please guide if hostname expected on line 10 is correct?
-  How do we compare dev, stage, and prod?

Reply:
I guess localhost could be considered dev